src/fetchtastic/download/cli_integration.py

```
@@ -601,6 +587,27 @@ def _convert_results_to_legacy_format(
+            if is_client_app:
+                file_path = getattr(result, "file_path", None)
+                download_url = getattr(result, "download_url", None)
+                file_name = (
+                    os.path.basename(str(file_path))
+                    if file_path
+                    else os.path.basename(str(download_url)) if download_url else ""
+                )
+                is_android = is_android or is_android_asset_name(file_name)
+                is_desktop = is_desktop or is_desktop_asset_name(file_name)
+                if not is_android and not is_desktop:
+                    log.debug("Unclassified client-app asset, defaulting to Android")
```

Ruff (0.15.12)

[error] Undefined name `log`

(F821)